aci/resource_aci_infrarsdomp.go

```diff
@@ -50,7 +50,7 @@ func getRemoteInfraRsDomP(client *client.Client, dn string) (*models.InfraRsDomP
 	}
 	infraRsDomP := models.InfraRsDomPFromContainer(infraRsDomPCont)
 	if infraRsDomP.DistinguishedName == "" {
-		return nil, fmt.Errorf("InfraRsDomP %s not found", dn)
+		return nil, fmt.Errorf("Infra RsDomP %s not found", dn)
```

Comment:
Should we reword RsDomP to something like Domain?

Reply:
I looked it up and couldn't find any logical term for it. Did you mean RsDomainP?

Reply:
Relationship to Domain Profile